plugins/global-filters.js

```diff
@@ -0,0 +1 @@
+import '../filters/datetime'
```

Comment:
Please read the card https://trello.com/c/Ak5saPN0/23-create-a-global-datetime-filter carefully?

> ## Plugins
> Created  `~plugins/global-filters.js` files that registers all global filters. it should register the date/time filter function under the `datetime`, `date` and `time` names

```suggestion
import Vue from 'vue'
import { datetime, date, time } from '../filters/datetime'

Vue.filter('datetime', datetime)
Vue.filter('date', date)
Vue.filter('time', time)
```